trunner/device.py

```diff
@@ -19,7 +30,27 @@
 }
 
 
-class DeviceRunner:
+def proccess_log_output(proc):
+    while True:
+        output = proc.stdout.readline().decode('utf-8')
+        if proc.poll() is not None and output == '':
+            break
+        if output:
+            logging.info(output)
+
+
+class Runner:
+    def boot(self):
```

Comment:
- please add one-sentence docstring regarding what the given function should do (so: describe the interface)
- i'm not sure if `boot` is not rt1064-specific, maybe `reboot` would be more desirable in different Runners (like Qemu)?

Reply:
1. I added docstrings
2. I removed `boot` from `Runner`. Maybe I'll put it into the `DeviceRunner` class in the future when more targets arrives.